src/components/progress/progress-bar.js

```diff
@@ -6,9 +6,7 @@ export default {
   inject: {
     progress: {
       from: 'progress',
-      default: function() {
-        return {}
-      }
+      default: () => {}
```

Comment:
I think this might need to be converted to:

```js
  default: () => ({})
```

So that an actual empty object is returned.